resources/js/pages/Signup.vue

```diff
@@ -0,0 +1,53 @@
+<script setup lang="ts"></script>
```

Comment:
- create a sub-directory named `auth`
- rename `Signup.vue` to `Register.vue`
- put the `Register.vue` in `auth` directory